src/Numeric/Interval/Kaucher.hs

```diff
@@ -76,6 +76,11 @@ import qualified Data.Semigroup
 import qualified Data.Monoid
 
 -- $setup
+-- eta expanded to avoid defaulting with older GHCi
```

Comment:
When you say "older GHCi", precisely which versions of GHC suffer from this? I was under the impression that this only mattered for GHC 7.0 and 7.2, and because we specify `docspec: >=7.6` elsewhere, I would have thought that we could get away without this eta expansion. But perhaps I'm missing something.

Reply:
yes, GHC-7.0. If not the properties (which need `default` which is only GHC-7.6), then that comment is still valid.

Reply:
Should I however remove that comment? I don't think that eta-expansion hurts either.

Reply:
> Should I however remove that comment?

I'm not sure I understand https://github.com/ekmett/intervals/pull/65/files#r554631769 correctly, but if we always run new enough versions to avoid defaulting issues, then I would advocate removing the comment _and_ eta contracting the definitions in the doctests below, since it makes them more resilient to changes in the future.